### PR TITLE
Add Skeleton settings to config and change some FK averaging logic.

### PR DESCRIPTION
**Pull request by Louka3000**
Protocol stuff is in another PR by butlah.
This PR:
- renames SkeletonConfigValue to SkeletonConfigOffsets
- Renames SkeletonConfigToggle to SkeletonConfigToggles
- Adds SkeletonConfigValues
- makes the extended pelvis model only work if user doesn’t have a hip tracker and make pelvis averaging roll and yaw only.

### CODE REVIEW

**Comment by Eirenliel** [comment]
#210 was merged, please rebase, fix conflicts and can mark for review.

**Comment by Louka3000** [comment]
> #210 was merged, please rebase, fix conflicts and can mark for review.

Still needs SolarXR PR

**Comment by TheButlah** [comment]
This was written berfore https://github.com/SlimeVR/SolarXR-Protocol/pull/31 I think, so it may require some revision

**Comment by Louka3000** [comment]
@TheButlah Will make the protocol stuff in another PR. This PR is ready to be merged.

File: src/main/java/dev/slimevr/gui/SkeletonConfigGUI.java [hunk 1]
```diff
@@ -93,8 +93,9 @@ float proportionsIncrement(Boolean negative) {
 		return increment;
 	}
 
-	String getBoneLengthString(SkeletonConfigValue joint) { // Rounded to the
-															// nearest 0.5
+	String getBoneLengthString(SkeletonConfigOffsets joint) { // Rounded to
+																// the
+		// nearest 0.5
```
Comment (ButterscotchV) [inline src/main/java/dev/slimevr/gui/SkeletonConfigGUI.java:98]: Lol what's up with this comment?

**Review by ButterscotchV — APPROVED** [review]

File: src/main/java/dev/slimevr/gui/SkeletonConfigGUI.java (hunk 1 above)
Comment (Louka3000) [reply src/main/java/dev/slimevr/gui/SkeletonConfigGUI.java:98]: automatic formatting go brrr.
Old GUI code will get deleted anyway and as you said (on Discord for anyone reading) it doesn’t matter so I’m gonna resolve this

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**